### PR TITLE
WRQ-27998: Updated so that driver path can be found in Chrome 114 or higher

### PR DESCRIPTION
**Pull request by mmyelyn**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] I have run automated testing and it is passed
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
We created a server for Chrome version 120, and an issue occurred where Chrome driver 120 version was not found when building on a server where Chrome version 120 was applied

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
We can set args in selenium-standalone service. Update the driver import path as it has changed since Chrome version 114
refer to : 
 - https://chromedriver.storage.googleapis.com/LATEST_RELEASE
 - https://developer.chrome.com/docs/chromedriver/downloads/version-selection

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-27998

### Comments
Enact-DCO-1.0-Signed-off-by: Hyelyn Kim (myelyn.kim@lge.com)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 dist: focal
 language: node_js
 node_js:
-    - node
+    - "21"
```
Comment (hong6316) [inline .travis.yml:4]: Should we need this?

Comment (mmyelyn) [reply .travis.yml:4]: As with enact and sandstone, there is a canvas issue when running the node as current(node 22). a travis error occurs.

Comment (hong6316) [reply .travis.yml:4]: Hmm, when I revert the version "21" to "node", travis error does not occurred
https://github.com/enactjs/ui-test-utils/commit/a0937978d8230f78a07b89c3456ac812be24e0c7

You don't need to use the version "21".

Comment (mmyelyn) [reply .travis.yml]: Ok, It's fixed, When I tested first. I looked at the travis results before merge develop.
(https://github.com/enactjs/ui-test-utils/compare/develop...test/chrome-driver)

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**